ADOPTERS.md

```diff
@@ -0,0 +1,55 @@
+
+# Adopters
+
+This document is not an official list, but is collated from a variety of
+confirmed and unconfirmed sources. If your organization is erroneously present,
+misspelled or miswritten in any way, please reach out to us right away and we
+will amend the list. Thank you for your understanding in this regard.
+
+## New Adopters
+
+If your company, studio, or organization uses rez, please consider filing a
+GitHub issue, or pull-request, asking us to add you.
+
+You are also encouraged to fill in and confirm or annotate any of the info in
+the unofficial [rez adopters spreadsheet](https://docs.google.com/spreadsheets/d/15IUtYIePvQOtIt7W639oOPLsl5qcvB_jJhoyHelhep8/edit#gid=0).
+
+Thank you help us to understand the rez ecosystem! It helps us understand how
+best to grow and support rez usage in workflows everywhere.
```

Comment:
I think there is a typo? Shouldn't this be something like "Thank you for helping us"?

I the second sentence i would likely have said "It helps us understand how to best grow". 

Mind you i am not a native speaker and i might be completely off.

Reply:
Good comment.. I've rewritten that chunk. https://github.com/AcademySoftwareFoundation/rez/pull/1720/commits/2e0457764f9d0347a58185a1e754fef6f52c5b1c